### ISSUE TITLE
where to add code to hide WCAG accessibility logo and link in xot template?

### ISSUE THREAD

**Issue by ronm123** [thexerteproject/xerteonlinetoolkits#941]
Hi all
In the Bootstrap code I added the WCAG Accessibility statement link and logo to the footer in rloObject.html, some options to hide the logo and/or change the alt text and link in the wizard and then the code to deal with all that in application.js all that seems to work fine.

I'd also previously added the WCAG logo and link to colourChanger.html for XOT projects and again I think that all works fine. However when I added this to the XOT template I hadn't thought about giving authors the options to hide the logo and link or change the link to their own etc. So I want to add those options to XOT similar to already added to Bootstrap but I'm not sure how best to do so. The code needed for the wizard is easy and the same as already in Bootstrap e.g.
`<wcagGroup type="group" label="Accessilibity logo and link" optional="true" tooltip="Hide the accessilibity logo and link or change the link.">
            <wcagHide label="Hide the Accessibility logo and link" type="Checkbox" conditionTrigger='true' defaultValue="false" optional="true"/>
			<wcagAlt label="Accessilibity logo alt text" type="TextInput" defaultValue="WCAG WAI-AA logo" condition="wcagHide=='false'" optional="true"/>
			<wcagLinkTitle label="Link tooltip" type="TextInput" defaultValue="View the Xerte accessibility statement" condition="wcagHide=='false'" optional="true"/>
            <wcagLink type="Media" label="Accessibility statement link" tooltip="Add a url or browse to a file to link to your own accessibility statement. Leave empty to link to the accessibility statement on xerte.org.uk." condition="wcagHide=='false'" optional="true" />
		</wcagGroup>`

I then looked at adding the conditional code to Xenith which would be something like the following:
`		//populate wcag logo and link and/or hide it
		if (x_params.wcagAlt != undefined){
			$(".wcagLogo").attr("alt",x_params.wcagAlt);
		}
		if (x_params.wcagLinkTitle != undefined){
			$(".wcagLink a").attr("title",x_params.wcagLinkTitle);
		}
		if (x_params.wcagLink != undefined && x_params.wcagLink != ""){
			$(".wcagLink a").prop("href",x_params.wcagLink);
		}
		if (x_params.wcagHide != undefined && x_params.wcagHide == 'true'){
			$('.wcagLink').remove();
		}`
But I'm not sure how best to then trigger that when the colour changer is opened or if the best thing to do is to put it in colourChanger.html how to access the xml params or attributes because there doesn't seem to be anything else in that file accessing the xml?

Can someone advise/help with this?

**Comment by torinfo** [comment]
@ronm123 Probably best to put it in colourChanger.html, because that should be easier than trigger xentith code when courChanger is opened, but also that option is doable. In colourChanger.html you can just use x_params. That's available in the model files as well.

**Comment by ronm123** [comment]
Thanks @torinfo I was just being stupid! ;-( Was checking for x_params values but was showing as undefined because I hadn't added the optional group to the project! Committed now https://github.com/thexerteproject/xerteonlinetoolkits/commit/2f1752af8a00d89488419ccb9def1f9f14948f00 so will close this issue.
